### PR TITLE
feat: k6-tests

### PR DESCRIPTION
**Pull request by jlangy**
refactor old tests, include introspection, user info, and offline sessions. Also add a podman-compose file to run our custom image locally, and a small electron app to track podman stats output.

I removed the old singleRealm and multiRealm tests, squashed them all into the activeSessions test and just made the number of realms configurable, they were pretty much duplicates.

### CODE REVIEW

File: k6/userInfo.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        synchronousExecutions: {
+            executor: 'per-vu-iterations',
+            vus: CONCURRENT_LOOPS,
+            iterations: ITERATIONS_PER_LOOP,
+        }
+    },
+    thresholds: {
+        http_req_failed: [
+            {
+                threshold: `rate<${MAX_ALLOWED_FAILURE_RATE}`,
```
Comment (thegentlemanphysicist) [inline k6/userInfo.js:20]: is the max allowed failure rated defined anywhere?

**Review by thegentlemanphysicist — COMMENTED** [review]
It looks like the max failure rate constant isn't defined in some of these scripts.

File: k6/userInfo.js (hunk 1 above)
Comment (jlangy) [reply k6/userInfo.js:21]: Added that change in with the new test now

File: k6/constantRateAllFlows.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    scenarios: {
+        peakProfile: {
+          executor: 'constant-arrival-rate',
+          duration: '2h',
+          timeUnit: '1m',
+          rate: 34,
+          preAllocatedVUs: 5,
+        },
+        // stress: {
+        //     executor: 'ramping-arrival-rate', //Assure load increase if the system slows
```
Comment (thegentlemanphysicist) [inline k6/constantRateAllFlows.js:25]: Are these uncommented for different tests? Or can we delete the comments?

**Review by thegentlemanphysicist — APPROVED** [review]

**Pull request merged**